packs/hubot/requirements.txt

```diff
@@ -1,2 +1,3 @@
 requests>=2.5.1,<2.6
 six
```

Comment:
Yeah, good catch.

Seems it's just global requirement: https://github.com/StackStorm/st2/blob/a41fc5d95134b98947c456ff6d2a49435fc6f035/requirements.txt#L16